revised.tex

```diff
@@ -0,0 +1,555 @@
+* Corresponding author. \\
+\bigskip
+{\ddag} These authors contributed equally to this work.
+\end{flushleft}
+
+\section*{Abstract}
+
+To survive and thrive,
+a community must attract and retain new members
+and help them be productive.
+The 10 simple rules in this paper describe ways to do this
+that are suitable for projects in open source, open science, and open education,
+and are based on both current research and direct experience.
+
+\section*{Author Summary}
+
+Research has always been a communal activity,
+but researchers have only recently begun to investigate why some communities thrive and others fail.
+Attracting contributors,
+retaining them,
+and helping them be more productive
+have all been the subject of intensive study.
+This paper presents 10 findings from those studies that community leaders can apply immediately
+and explains why they work.
+
+\section*{Introduction}
+
+To survive and thrive,
+a community must attract new members,
+retain them,
+and help them be productive~\cite{qureshi2011}.
+As openness becomes the norm in research, software development, and education,
+knowing how to do this has become a essential skill
+for principal investigators and community managers alike.
+A growing body of knowledge in sociology, anthropology, education, and software engineering
+can guide decisions about how to facilitate this.
+
+What exactly do we mean by ``community''?
+In the case of open source and open science,
+the most usual meaning is a ``community of practice''.
+As defined by Lave and Wenger~\cite{lave1991,wenger1999},
+groups as diverse as knitting circles, oncology researchers, and web designers
+share three key characteristics:
+
+\begin{enumerate}
+
+\item Participants have a common product or purpose that they work on or toward.
+
+\item They are mutually engaged, \textit{i.e.}, they assist and mentor each another.
+
+\item They develop shared resources and domain knowledge.
+
+\end{enumerate}
+
+Brown~\cite{brown2019} specializes this to define a ``community of effort'' as,
+``\textit{{\ldots}a community formed in pursuit of a common goal.
+The goal can be definite or indefinite in time,
+and may not be clearly defined,
+but it is something that (generally speaking) the community is aligned on.}''
+People working to preserve coral reefs in the face of global climate change
+are an example of such a community.
+No central organization coordinates their work,
+but the scientists who study coral reefs,
+the environmentalists who work to protect them,
+and the citizens who support them financially and politically
+are aware of each other's efforts,
+collaborate in ad hoc ways,
+and are conscious of contributing toward a shared purpose.
+
+Open source software projects are also communities of effort.
+For example,
+the Mozilla Firefox~\cite{mozilla} community includes a mix of paid professionals,
+highly-involved volunteers,
+and occasional contributors who create software,
```

Comment:
I'd give this either a _both-and_ construct or a _not only-but_ construct.
> who not only create software, documentation, and tutorials, but also…

or

> who create both software, documentation, and tutorials, as well as organize events…

Reply:
:+1: